web/pdf_viewer.js

```diff
@@ -138,6 +138,8 @@ function isValidAnnotationEditorMode(mode) {
  *   The default value is `true`.
  * @property {boolean} [enableAutoLinking] - Enable creation of hyperlinks from
  *   text that look like URLs. The default value is `true`.
+ * @property {number} [minDurationToUpdateCanvas] - Minimum duration to wait
+ *  before updating the canvas.
```

Comment:
```suggestion
 *   before updating the canvas.
```

Reply:
Please also mentioned the default value in the comment, similar to the other options.